traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 8.9.2
+version: 8.9.3
```

Comment:
Thanks. Now, last step (sorry) - its adds a new functionality and thus needs to be a minor bump on the version :)

Reply:
@SantoDE no prob. Bumped to 8.11.0.

But shouldn't it be 8.10?